src/commons_codec/transform/dynamodb.py

```diff
@@ -132,13 +134,18 @@ def decode_record(self, item: t.Dict[str, t.Any]) -> DualRecord:
 
 
 class DynamoDBFullLoadTranslator(DynamoTranslatorBase):
-    def to_sql(self, record: t.Dict[str, t.Any]) -> SQLOperation:
+    def to_sql(self, data: t.Union[RecordType, t.List[RecordType]]) -> SQLOperation:
         """
         Produce INSERT|UPDATE|DELETE SQL statement from INSERT|MODIFY|REMOVE CDC event record.
         """
-        dual_record = self.decode_record(record)
         sql = f"INSERT INTO {self.table_name} ({self.TYPED_COLUMN}, {self.UNTYPED_COLUMN}) VALUES (:typed, :untyped);"
-        return SQLOperation(sql, {"typed": dual_record.typed, "untyped": dual_record.untyped})
+        if not isinstance(data, list):
+            data = [data]
+        parameters = []
+        for record in data:
+            parameter = self.decode_record(record).to_dict()
+            parameters.append(parameter)
+        return SQLOperation(sql, parameters)
```

Comment:
Any chance to improve this, by using a construct around `map`, or a list comprehension? Is it worth the performance improvements / memory savings?

Reply:
Using a list comprehension now.